crates/sui-protocol-config/src/snapshots/sui_protocol_config__test__Mainnet_version_13.snap

```diff
@@ -0,0 +1,182 @@
+---
+source: crates/sui-protocol-config/src/lib.rs
```

Comment:
Can you remove this file and the `Testnet` one - these files were generated by the tests in main which I recently changed to have per-chain snapshots. But the test changes aren't in the release branch yet, so these files are vestigial (and may cause confusion if a later change is picked in that does include them).